src/local-time.lisp

```diff
@@ -397,6 +424,47 @@
     (error "Seems like the timezone repository has not yet been loaded. Hint: see REREAD-TIMEZONE-REPOSITORY."))
   (gethash name *location-name->timezone*))
 
+(defun find-timezone-by-abbreviated-subzone-name (abbreviated-name &key unix-time timestamp)
+  "Returns one of the active timezones, matched subzone and last transition time
```

Comment:
This function's interface would be more clear if it only returned a list of matches at the current time. Another function could return the historical/future matches.  Also, since this is an exported interface, it should use a timestamp instead of unix-time. Maybe like this:

- `TIMEZONES-MATCHING-SUBZONE abbreviated-name timestamp` - returns matching subzones that are used at the current time in the form (zone subzone last-transition-time).
- `ALL-TIMEZONES-MATCHING-SUBZONE abbreviated-name` - returns all subzones with the matching abbreviated name in the form (zone subzone last-transition-time)

Callers that want a single zone can just use `FIRST` or `RANDOM-ELT`.

Reply:
Makes sense. Done